pkg/aggregator/demultiplexer.go

```diff
@@ -110,10 +113,11 @@ func DefaultDemultiplexerOptions(options *forwarder.Options) DemultiplexerOption
 	}
 
 	return DemultiplexerOptions{
-		SharedForwarderOptions:    options,
-		FlushInterval:             DefaultFlushInterval,
-		UseEventPlatformForwarder: true,
-		UseOrchestratorForwarder:  true,
+		SharedForwarderOptions:         options,
+		FlushInterval:                  DefaultFlushInterval,
+		UseEventPlatformForwarder:      true,
+		UseOrchestratorForwarder:       true,
+		UseContainerLifecycleForwarder: true,
```

Comment:
This set to `true` means it will be enabled for all binaries (serverless, infra ageet, security, ...).

Should we set this to `false` and only set it to `true` while instanciating the infra agent in `cmd/agent/app/run.go`?

Reply:
agreed 👍